Soccer Match.rtf

```diff
@@ -7,12 +7,29 @@
 
 \f0\fs24 \cf0 The app we\'92re building is soccer game. we need 2 teams of 11 players. they will play in a match. the match will keep the score. at the end we will display who wins.\
 \
+
+const Team = [
+  {
+  name: "FC Barcelona",
+  position: 1,
+  isChampion: false  
+},
+  {
+  name: "Real Madrid",
+  position: 2,
+  isChampion: false
+}
```

Comment:
is this how you create an object in Javascript (where its properties' values are inferred rather than explicitly declared)?  i'd recommend these properties:

```
name: "Blah"
players: [Player(name: "Vini Jr", goals: 0, country: "Brazil", age: 28, team: blah, position: "striker")]
leagueStanding: 1
```